lib/attache/download.rb

```diff
@@ -48,7 +48,7 @@ def _call(env, config)
           nil
         end
 
-        unless file
+        unless file && file.try(:size).to_i > 0
```

Comment:
Turns out `activesupport` was brought in via paperclip dependencies. But I'll add explicit requirement